rest/admin_api_test.go

```diff
@@ -1328,10 +1331,17 @@ func TestResyncErrorScenarios(t *testing.T) {
 	response = rt.SendAdminRequest("POST", "/db/_resync?action=start", "")
 	assertStatus(t, response, http.StatusOK)
 
-	// Unlikely but this may cause a race as we are expecting this POST to run before the 1000s doc re-sync occurs.
-	// May require a method to slowdown the resync process
-	response = rt.SendAdminRequest("POST", "/db/_resync?action=start", "")
-	assertStatus(t, response, http.StatusServiceUnavailable)
+	if base.TestsDisableGSI() {
```

Comment:
As discussed, looks like these should be getting set before running the resync on line 1331 to avoid race.